src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -128,7 +128,10 @@
             "http://hl7.org/fhir/SearchParameter/individual-given",
             "http://hl7.org/fhir/SearchParameter/individual-birthdate",
             "http://hl7.org/fhir/SearchParameter/clinical-date"
-        ]
+        ],
+        "ParallelQueryOptions": {
+            "MaxQueryConcurrency": 1000
```

Comment:
Should we have default value as 500?

Reply:
Agree, for now, lets use 500. 
I think its also fine for now for this to be the hardcoded default because this is the best known current value. This gives us the option to change at a later point without recompile.

Reply:
Thanks, I have defaulted it to 500. I understand for 1000 it is taking much longer, but what about setting it to 750 by default. 
Just another thought - As per the test numbers shared by Narasi, I see that one of the occurence of 500 took 27secs and 750 took 28secs, so I think 750 is still better.

![image](https://user-images.githubusercontent.com/57157506/122128075-fe732d80-cde8-11eb-948c-354f0a3570f4.png)

Reply:
The ticket @namadabu opened confirmed that this is a CPU/Task limitation (https://github.com/Azure/azure-cosmos-dotnet-v3/issues/2556) I think we should stick with a lower number. It is configurable so we can alter later in higher perf environments

Reply:
Makes sense, thanks!